server/models/languages.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+// Possibly incorrect usage of enum here
+const languagesSchema = new mongoose.Schema({
+  languages: [
+    {
+      enum: [
+        "Python",
+        "JavaScript",
+        "Java",
+        "PHP",
+        "GoLang",
+        "C#",
+        "C++",
+        "Ruby",
+      ],
+      type: String,
+    },
+  ],
```

Comment:
Should we assume all languages exist on one document, or should each language be a separate document that can be embedded within `User.languages`?